packages/gatsby-transformer-remark/src/create-schema-customization.js

```diff
@@ -40,7 +40,9 @@ module.exports = (nodeApiArgs, pluginOptions = {}) => {
   // to customize the GraphQL schema. This makes it possible for subplugins to
   // modify types owned by `gatsby-transformer-remark`.
   plugins.forEach(plugin => {
-    const resolvedPlugin = require(plugin.resolve)
+    const resolvedPlugin =
+      _CFLAGS_.GATSBY_MAJOR === `4` ? plugin.module : require(plugin.resolve)
```

Comment:
So the old behavior will be incompatible with v4 and will be a breaking change? Did we consider keeping it compatible? I am fine with both but if it will be a breaking change we should probably add a note in https://github.com/gatsbyjs/gatsby/discussions/32860

Reply:
`require(plugin.resolve)` will continue to work in V4, but it will be undesirable to use it due to bundling problems with dynamic imports and using `.subPlugins` will be recomended. Not sure if there is nice way to "deprecate" old style here